README.md

```diff
@@ -814,8 +814,6 @@ Here is a list of template creators:
    * C# (.NET Standard 1.3 ): @Gronsak
    * C# (.NET 4.5 refactored): @jimschubert [:heart:](https://www.patreon.com/jimschubert)
    * Clojure: @xhh
-   * Dart: @yissachar
-   * Dart (refactor): @joernahrens
```

Comment:
I suggest we keep these lines to give credits to those who create/refactor the Dart 1.x generator in which the Dart 2.x generator is based on if I'm not mistaken.

Reply:
Done